cdip_admin/api/v2/serializers.py

```diff
@@ -1010,3 +1020,53 @@ class ActivityLogRetrieveSerializer(serializers.Serializer):
 class ActionTriggerSerializer(serializers.Serializer):
     run_in_background = serializers.BooleanField(required=False, default=False)
     config_overrides = serializers.JSONField(required=False, write_only=True)
+
+
+class UserAgreementSerializer(serializers.ModelSerializer):
+    user = serializers.HiddenField(default=serializers.CurrentUserDefault())
+    eula = serializers.HiddenField(default=EULA.objects.get_active_eula)
+    accept = serializers.BooleanField(read_only=True)
+    date_accepted = serializers.DateTimeField(read_only=True)
+
+    class Meta:
+        model = UserAgreement
+        fields = (
+            "user",
+            "eula",
+            "accept",
+            "date_accepted",
+        )
+
+    def get_unique_together_validators(self):
+        # Overriden to disable unique together check as it's handled in the create method
+        return []
```

Comment:
nitpick: do we have a return type like not implemented? (not sure if it's an option)

Reply:
Thanks for the suggestion. If the base class was part of our code, we could consider raising `NotImplementedError` or handling that case in other ways. But here we are overriding a method implemented in a base class that is part of the `django-rest-framework` so we can't / don't want to change how that works. Here I wanted to disable some validators, and the way of doing that with DRF is to return that empty list.